app/models/manageiq/providers/base_manager/event_catcher/runner.rb

```diff
@@ -84,8 +125,8 @@ def monitor_events
     raise NotImplementedError, "must be implemented in subclass"
   end
 
-  def process_event(_event)
-    raise NotImplementedError, "must be implemented in subclass"
+  def process_event(event)
+    queue_event(event) unless filtered?(event)
```

Comment:
How about adding a `filtered?` method to the base class that just returns false?
This way we don't enforce a `filtered?` to be implemented in all subclasses